File: sprint03/KSH/PGS_42578.py
def solution(clothes):
    answer = 1
    
    a = dict()
    num_lst = []
    
    for i in clothes:
        if a.get(i[1]) == None:
            a[i[1]] = 1
        else:
            a[i[1]] += 1
        
    for i in a:
        answer *= (a[i] + 1)
        
    return answer - 1

# 조합을 이용해서는... 푸는게 아니다.
# 이건 그냥 수학 문제 => 확률과 통계를 공부했는가?

File: sprint03/JMS/PGS_42578.cpp
#include <string>
#include <vector>
#include <unordered_map>

using namespace std;

unordered_map<string, int> map;

int solution(vector<vector<string>> clothes) {
    int answer = 1;

    for(auto p : clothes)
        map[p[1]]++;

    for(auto p : map)
        answer *= (p.second + 1);
    
    return answer - 1;
}
